SpatialGDK/Source/SpatialGDKEditor/Private/Utils/TransientUObjectEditor.cpp

```diff
@@ -0,0 +1,167 @@
+#include "Utils/TransientUObjectEditor.h"
+
+#include "PropertyEditor/Public/PropertyEditorModule.h"
+#include "MainFrame/Public/Interfaces/IMainFrameModule.h"
+
+#include "SlateApplication.h"
```

Comment:
Please order includes

Reply:
What is the order ?

Reply:
Just alphabetical. `SB` before `Sl` in this case.

Reply:
Not sure I see the point of this ordering, but OK